src/components/ImagesList/ImagesList.js

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect } from 'react';
+import { useImageListStyle } from './ImagesList.style';
+import ImageList from '@material-ui/core/ImageList';
+import ImageListItem from '@material-ui/core/ImageListItem';
+import ImageListItemBar from '@material-ui/core/ImageListItemBar';
+import Box from '@material-ui/core/Box';
+import IconButton from '@material-ui/core/IconButton';
+import GetAppRoundedIcon from '@material-ui/icons/GetAppRounded';
+import axios from "axios";
+import ThumbUpAltRoundedIcon from '@material-ui/icons/ThumbUpAltRounded';
+import { useSelector, useDispatch } from 'react-redux';
+import { getLikedPhotosId } from '../../actions';
+import { connect } from 'react-redux';
+import { imageSelector } from '../../selectors';
+
+
+const ImagesList = props => {
+  const classes = useImageListStyle();
+  const { images } = props;
+  const myState = useSelector((state) => state.authReducer);
```

Comment:
We should declare selector using re-select library that caches the state tree value. Selector call like this serves no benefit to us